build/Dockerfile

```diff
@@ -1,17 +1,37 @@
+## BUILDTIME ##
```

Comment:
Ugh. I like the comment, but boilerplate is currently [hardcoded to expect the initial `FROM` to be on the first line](https://github.com/openshift/boilerplate/blob/30308eb5c5b225744daa3d2d697a9bff92dcf34b/boilerplate/openshift/golang-osd-operator/update#L16). Can you (re)move the comment for now, and maybe we can work on making the bp logic a bit smarter?

Reply:
> making the bp logic a bit smarter?

Perhaps we can use something like https://github.com/containerbuildsystem/dockerfile-parse